### ISSUE TITLE
Incorrect dependency versions specified are causing warnings

### ISSUE THREAD

**Issue by mikesprague** [carloscuesta/gitmoji-cli#801]
Hello @carloscuesta!

Issue:

When upgrading my global install to the latest version (`npm i -g gitmoji-cli`) I received some dependency and peer dependency warnings about the versions requested. This has happened everywhere I use it so far (MacOS, Ubuntu via WSL2, and inside of GitHub Codespaces).

**OS**:  MacOS (and Ubuntu via WSL2)
**gitmoji -v**: `4.10.0`
**Node version**: `v17.7.2` (and `v16.14.2`)

**Error Message**:

```shell
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: inquirer-autocomplete-prompt@1.0.1
npm WARN Found: inquirer@8.1.1
npm WARN node_modules/gitmoji-cli/node_modules/inquirer
npm WARN   inquirer@"8.1.1" from gitmoji-cli@4.10.0
npm WARN   node_modules/gitmoji-cli
npm WARN     gitmoji-cli@"*" from the root project
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer inquirer@"^5.0.0 || ^6.0.0" from inquirer-autocomplete-prompt@1.0.1
npm WARN node_modules/gitmoji-cli/node_modules/inquirer-autocomplete-prompt
npm WARN   inquirer-autocomplete-prompt@"1.0.1" from gitmoji-cli@4.10.0
npm WARN   node_modules/gitmoji-cli
npm WARN
npm WARN Conflicting peer dependency: inquirer@6.5.2
npm WARN node_modules/inquirer
npm WARN   peer inquirer@"^5.0.0 || ^6.0.0" from inquirer-autocomplete-prompt@1.0.1
npm WARN   node_modules/gitmoji-cli/node_modules/inquirer-autocomplete-prompt
npm WARN     inquirer-autocomplete-prompt@"1.0.1" from gitmoji-cli@4.10.0
npm WARN     node_modules/gitmoji-cli
```

<img width="854" alt="Screen Shot 2022-03-21 at 9 50 12 AM" src="https://user-images.githubusercontent.com/560705/159275130-66624b09-2150-4ac1-9662-226e547e1626.png">

Please let me know if there's any additional information that would be useful. Thanks!

**Comment by carloscuesta** [comment]
Hey @mikesprague thanks for opening the issue,

You can ignore the warning, the `cli` should work in the same way as of now, I opened a PR to update the dependency so the warning doesn't appears anymore when installing the cli.

Will publish an update soon 😊

**Comment by carloscuesta** [comment]
Fixed and published in `4.11.0` 🙏🏼 
<img width="1056" alt="Screen Shot 2022-03-21 at 18 22 41" src="https://user-images.githubusercontent.com/7629661/159329218-20c4bd71-dd39-497e-bfc8-4ce7b3aa279c.png">
